### DISCUSSION TITLE
are all icons supported in wave 0.20.0

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1401] [Q&A] [answered]
is it expected that some icon(s) on the page https://wave.h2o.ai/docs/icons are not supported/rendered in an app built with Wave v0.20.0?

Ex: StorageAcount

**Comment by mturoci** [comment, marked as answer]
Yes, expected. [https://wave.h2o.ai/docs/icons](https://wave.h2o.ai/docs/icons) are for 0.21 and onwards, [https://uifabricicons.azurewebsites.net/](https://uifabricicons.azurewebsites.net/) are for 0.20 and older.

**Reply by azim-b** [reply]
thanks - but I suppose not all icons on https://uifabricicons.azurewebsites.net/ are open source?

**Reply by mturoci** [reply]
I guess they are open-source, but have different license than the ones in 0.21

**Reply by azim-b** [reply]
so it's ok to use any of these in an app using v0.20.0?

**Reply by mturoci** [reply]
Ideally, you should use 0.21, but icons sin 0.20 should be fine as well as long as you do not use any Microsoft-specific logos without their explicit approval.
